packages/components/button/src/ButtonGroup/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { SpacingTokens } from '@contentful/f36-tokens';
+
+export type ButtonGroupVariants = 'separate' | 'collapse';
```

Comment:
a small thing, but I'd rather use adjectives instead of verbs here:

variant="collapsed" sounds more logical than variant="collapse"
variant="separated" (or even "spaced" because we have "spacing" property)